coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorServer.java

```diff
@@ -108,6 +109,12 @@ private void initialization() throws FileNotFoundException {
         new AssignmentStrategyFactory(coordinatorConf, clusterManager);
     this.assignmentStrategy = assignmentStrategyFactory.getAssignmentStrategy();
 
+    if (coordinatorConf.getBoolean(CoordinatorConf.COORDINATOR_ACCESS_MANAGER_ENABLE, false)) {
```

Comment:
AccessManager should be always initialized, and the checker can be empty

Reply:
Use empty checkers configuration to disable AccessManager?  No need to create an AccessManager instance if there is no checkers.